ui/DesignSystem/Primitive/Button.svelte

```diff
@@ -155,22 +155,123 @@
   }
 
   .disabled {
-    background-color: var(--color-foreground-level-3);
-    border-color: var(--color-foreground-level-3);
-    color: var(--color-foreground-level-5);
+    background-color: var(--color-foreground-level-1);
+    border-color: var(--color-foreground-level-1);
+    color: var(--color-foreground-level-3);
+    cursor: default;
   }
 
   .disabled:hover {
-    background-color: var(--color-foreground-level-3);
+    background-color: var(--color-foreground-level-1);
     box-shadow: 0 0 0 0;
-    border-color: var(--color-foreground-level-3);
-    color: var(--color-foreground-level-5);
+    border-color: var(--color-foreground-level-1);
+    color: var(--color-foreground-level-3);
   }
 
   .disabled:active {
-    background-color: var(--color-foreground-level-3);
-    border-color: var(--color-foreground-level-3);
-    color: var(--color-foreground-level-5);
+    background-color: var(--color-foreground-level-1);
+    border-color: var(--color-foreground-level-1);
+    color: var(--color-foreground-level-3);
+  }
+
+  .disabled :global(svg) {
```

Comment:
We probably should give the button a class and nest the global CSS under that class, just to make sure it doesn't bleed into other components.

Reply:
good point. Added a `.button` class in general. And then specified most "global styles" under the `.vanilla` style since that's kinda the default. I'm wondering if we should change the variant to `vanilla` instead of `primary`.

Reply:
>  I'm wondering if we should change the variant to vanilla instead of primary.

👍 

Would also be nice to have the `.vanilla` classes ordered in one place.